test/main.jl

```diff
@@ -26,4 +26,28 @@
             end
         end
     end
+
+    @testset "successful run GitLab" begin
+        mktempdir() do tmpdir
+            cd(tmpdir)
```

Comment:
Should probably just match what I did here,

https://github.com/JuliaRegistries/CompatHelper.jl/pull/350/files#diff-21e3ffd1ceccd2a142495277ae65b3e5bd5569c7ef00b4573757281e90bb2a11L10

Reply:
I think overall there is a bit of an issue with the tests `cd`ing into tmp dirs and then just staying there after the tempdirs disappear. I think the tests need a bit of an overhaul to clean that up a bit.

Reply:
It's more-so that we don't `cd(original_dir)` here that was causing me issues with integration testing afterwards.

I did clean up *some* of these in my PR.